rails_event_store_active_record-legacy/lib/rails_event_store_active_record/legacy/generators/templates/v1_v2_migration_template.rb

```diff
@@ -71,6 +70,7 @@ def up
       end
       add_index :event_store_events, :id, unique: true if rails_42
       add_index :event_store_events, :created_at
+      add_index :event_store_events, :event_type
```

Comment:
Isn't it redundant? We don't remove existing index from legacy schema and add a new one on the same column?

Reply:
Oh, sqlite...